hazelcast-jet-core/src/main/java/com/hazelcast/jet/impl/connector/HazelcastWriters.java

```diff
@@ -152,6 +157,33 @@ public static ProcessorMetaSupplier writeListSupplier(@Nonnull String name, @Nul
         ));
     }
 
+    public static ProcessorMetaSupplier writeObservableSupplier(@Nonnull String name) {
```

Comment:
can we add a normal reliable topic sink here as well?

Reply:
yes, sure, will do